apps/charterafrica/src/components/RichText/RichText.js

```diff
@@ -31,7 +27,10 @@ const serialize = (children, props) =>
     if (!node) {
       return null;
     }
-
+    const DEFAULT_PROPS = {
+      html: false,
+      ...props,
+    };
```

Comment:
Nope...

1. `DEFAULT_PROPS` is a `const` hence caps.
2. By spreading both `DEFAULT_PROPS` followed by `props` on component, we achieve the same thing without having to create a throwaway object.